src/v0/destinations/tiktok_ads/transform.js

```diff
@@ -129,10 +130,12 @@ const getTrackResponse = (message, Config, event) => {
 
 const trackResponseBuilder = async (message, { Config }) => {
   const { eventsToStandard, sendCustomEvents } = Config;
-  if (!message.event || typeof message.event !== 'string') {
-    throw new InstrumentationError('Either event name is not present or it is not a string');
+  validateEventName(message.event);
+  let event = message.event.toLowerCase().trim();
+  if (!event) {
+    throw new InstrumentationError('Event name is required');
```

Comment:
Shouldnt validateEventName take care of it?

Reply:
In case the event name is an empty string it would not take care of it